2022q2/clusteradm.adoc

```diff
@@ -0,0 +1,27 @@
+=== Cluster Administration Team
+
+Links: +
+link:https://www.freebsd.org/administration/#t-clusteradm[Cluster Administration Team members] URL: link:https://www.freebsd.org/administration/#t-clusteradm[https://www.freebsd.org/administration/#t-clusteradm]
+
+Contact: Cluster Administration Team <clusteradm@FreeBSD.org>
+
+FreeBSD Cluster Administration Team members are responsible for managing the machines the Project relies on to synchronise its distributed work and communications.
+In this quarter, the team has worked on the following:
+
+* Installed a new mirror in Sydney, Australia hosted by IX Australia
+* Fixed CI cluster hardware failure
+* Set up a new internal monitoring system
+
+Work in progress:
+
+* Work with the PowerPC team to improve the package builders, universal, and reference machines.
+* Plan Hardware refresh, and fixing misc failures in each sites
+* Improve the package building infrastructure
+* Review the service jails and service administrators operation
+* Working with doceng@ to improve https://www.freebsd.org and https://docs.freebsd.org
+* Improve the web service architecture
+* Improve the cluster backup plan
+* Improve the log analysis system
+
+We are in particular searching for possible hosting providers in Europe for setting up a new mirror.
+Please contact us if there is any possibilities for fitting the requirements for a link:https://wiki.freebsd.org/Teams/clusteradm/generic-mirror-layout[generic mirrored layout] or a link:https://wiki.freebsd.org/Teams/clusteradm/tiny-mirror[tiny mirror].
```

Comment:
I would suggest rewriting this paragraph:


We are looking for an additional full mirror site (five servers) in Europe.  See link:https://wiki.freebsd.org/Teams/clusteradm/generic-mirror-layout[generic mirrored layout] for our needs.  Offers of additional single-server mirrors (see link:https://wiki.freebsd.org/Teams/clusteradm/tiny-mirror[tiny mirror]) are always welcome too, especially in Europe.